budget_allocation/models/base_budget_move.py

```diff
@@ -75,7 +79,13 @@ def check_budget_allocation_limit(self, doclines):
                     )
                 )
                 continue
-            total_spend = sum([x["amount"] for x in source_fund_dict])
+            total_spend = sum(
+                [
+                    x["amount"]
```

Comment:
can we just use, `x.get("amount", 0.0)`? Not sure what amount could be?

Reply:
if query from budget.monitor.report, amount can be `None`